books/free-programming-books-et.md

```diff
@@ -35,9 +35,9 @@
 * [Javascript](http://lepo.it.da.ut.ee/~epuman/javascript.html) - Ella Puman, Tartu ÜLikooli õppematerjal
 
 
-##### Angular
+##### Angular.Js
```

Comment:
capital 'J'?

Reply:
Yep. I misktake.

Thanks for your eyes :hugs: